tools/visual-diff-ci/run_ci_script.py

```
@@ -214,6 +214,19 @@ def run_playwright():
     # try:
     run_command([NPM_PATH, "ci"], cwd=PLAYWRIGHT_DIR)
     run_command([NPX_PATH, "playwright", "install", "--with-deps"], cwd=PLAYWRIGHT_DIR)
+    # Node 24.16 + Percy's bundled extract-zip (yauzl) hangs while unpacking
+    # Percy's OWN Chromium: the download finishes, then extraction stalls forever
+    # (no Percy release fixes it — extract-zip 2.0.1 is the latest). Point Percy
+    # at the Chromium that Playwright (>=1.60) already installed and extracted, so
+    # Percy skips its own download/extract path entirely.
+    node = shutil.which("node")
+    chromium_exe = subprocess.check_output(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)